### DISCUSSION TITLE
COCOAリポジトリのアーカイブ化について

### DISCUSSION THREAD

**Discussion by keiji** [cocoa-mhlw/cocoa#1204] [General]
## 概要
本リポジトリについては今後アーカイブ（読み込み専用）モードに移行します。
また、アーカイブモードへの移行後、本リポジトリ（cocoa-mhlw/cocoa）は永年保存の取り扱いになります。

アーカイブモードへ移行する具体的な日程については、調整の上、追ってご報告します。

## Issueの整理
アーカイブモードへ移行する前に、Issueの整理を行います。
具体的には再度すべてのIssueを確認して、対応が終わっているものでclose漏れがあればcloseします。

## 永年保存とする背景
本リポジトリを永年保存することとなった背景は次のとおりです。

委託業者からの納品物にソースコードが含まれており、そこにGitHubリポジトリへの参照が含まれている。そのためGitHubリポジトリ自体も納品物と同期間保存する必要がある。

結核感染症課の保存期間表によれば、契約における納品物の保存期間は5年間だが、新型コロナ関係文書は一律で、保存期間満了後に国立公文書館に移管され、永年保存されることになっている。

以上のことから、COCOAのGitHubリポジトリも新型コロナ関係文書となり永年保存の取り扱いとなる。

### 参考
行政文書の管理における「歴史的緊急事態」の決定に伴う連絡会議 議事次第
https://www8.cao.go.jp/chosei/koubun/rekiren/haifusiryo.pdf

**Comment by keiji** [comment]
このリポジトリは4月中旬にアーカイブ化（Read-onlyへの切り替え）を予定しています。

**Reply by kvaluation** [reply]
ご報告書やデータ等を拝見してお礼や指摘などコメントすべきかどうか悩んでおりましたが、追加のコメントはこのリポジトリにはしないこととしました。アーカイブ化のご作業、よろしくおねがいします。
いままで、様々な情報発信、ありがとうございました。

**Reply by daisuke-nogami** [reply]
もしコメントがあるのであれば、ぜひ残していただけますと幸いです。
（GitHubレポジトリがArchive化されてしまいますと、行政に対するメッセージを出す手段が限定的になってしまいますので…）

**Comment by daisuke-nogami** [comment]
アーカイブ化作業は、今の所、明日（4/12）午後に実施予定です。
予定が変更になった場合には、また別途お知らせいたします。
